grpc/core/src/main/java/io/helidon/grpc/core/GrpcTracingContext.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2022, 2024 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.grpc.core;
+
+import java.util.Optional;
+
+import io.helidon.tracing.Span;
+
+import io.grpc.Context;
+
+/**
+ * Contextual information related to Tracing.
+ */
+public final class GrpcTracingContext {
```

Comment:
Is tracing built-into grpc? 
We have removed it from Helidon WebServer and I would expect we do the same thing for grpc - i.e. tracing is a standalone module that can be added.

Reply:
Moved tracing to its own module